k8s/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: palworld-secrets
+type: Opaque
+stringData:
+  rconPassword: yourRconPassword
```

Comment:
"yourRconPassword in base64 format" ?

Reply:
No, it's plaintext in the manifest files. It's base64 if you `kubectl describe` though.

I usually call it something like `example.secret.yaml`, make a copy, put the actual password in, apply the yaml with `kubectl apply -f secrets.yaml` then `rm secrets.yaml`.